elements/pfe-sass/mixins/_custom-properties.scss

```diff
@@ -0,0 +1,172 @@
+// New broadcast mixin, for use with content components
+//  1st arg:  lookup surface value from theme
+//  2nd arg:  list of broadcast variable - optional
+//  3rd arg:  identify the color set, default is surface - optional
+//
+//  usage:
+//    :host {
+//      #{pfe-set-broadcast(dark)};
+//    }
+//  result:
+//    :host {
+//      --pfe-broadcasted--color--text:            #{pfe-color(text--on-dark)};
+//      --pfe-broadcasted--color--link:            #{pfe-color(link--on-dark)};
+//      --pfe-broadcasted--color--link--visited:   #{pfe-color(link--visited--on-dark)};
+//      --pfe-broadcasted--color--link--hover:     #{pfe-color(link--hover--on-dark)};
+//      --pfe-broadcasted--color--link--focus:     #{pfe-color(link--focus--on-dark)};
+//    }
+
+@mixin pfe-set-broadcast($theme, $broadcast: $BROADCAST-VARS) {
+    @each $type in $broadcast {
+        $list: ();
+        $string: "";
+        $context: "";
+        $state: "";
+
+        @if $type == "link" {
+            $list: append($list, "#{$type}");
+
+            @each $s in (hover, focus, visited) {
+                $state: "--#{$s}";
+                $string: "#{$type}#{$state}";
+
+                @if not index($list, $string) {
+                    $list: append($list, "#{$type}#{$state}");
+                }
+            }
+        }
+        @else {
+            @if not index($list, $string) {
+                $list: append($list, "#{$type}");
+            }
+        }
+
+        @if index($THEMES, $theme) != null {
+            @if $theme != "light" {
+                $context: "--on-#{$theme}";
+            }
+        }
+        
+        @each $item in $list {
+            @if not index($BROADCAST-VARS, $type) {
+                @error "--#{$repo}-broadcasted--color--#{$item} variable is not currently supported.";
```

Comment:
```suggestion
                @error "--#{$repo}-broadcasted--color--#{$item} variable is not in the $BROADCAST-VARS list.
```

Reply:
actually i might have this wrong, what is it checking against?

Reply:
That's what it's checking against, yes. 👍